src/app.js

```diff
@@ -1 +1,26 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const [,, sourceFile, destinationFile] = process.argv;
+
+switch (true) {
```

Comment:
Switch true - bad case
Better move it if/else statement or to object (key it's condition and value it's action)